### ISSUE TITLE
[REQUEST] Simple Scanline Filter

### ISSUE THREAD

**Issue by JafuOleanders** [ekeeke/Genesis-Plus-GX#193]
So i noticed the newest Snes9xGX versions have a simple Scanline Filter added which looks really nice on EDTV and Plasma monitors, even on Modern LCD/LED. If possible, could such a Filter be added in the future?

**Comment by ekeeke** [comment]
Maybe.

**Comment by KMFDManic** [comment]
Filters can technically be done via RetroArch, itself, quite easily.  You just need the proper shader in the correct directory, then can load it directly via RetroArch, and also use it with most other Cores, rather than be tethered to just one.

**Comment by ekeeke** [comment]
There is no shader on GC/Wii and this is a request for stand-alone version, not retroarch version

**Comment by Edu789** [comment]
Is it possible to add filters for audio, video and screen options of the retroarch version?

**Comment by JafuOleanders** [comment]
> Is it possible to add filters for audio, video and screen options of the retroarch version?

Yes, Retroarch can do this, either by using the various shaders or with an overlay.

I guess it wasn't possible to add to the wii standalone… bummer.
